tests/modules/services/screen-locker/basic-configuration.nix

```diff
@@ -5,6 +5,7 @@
     enable = true;
     inactiveInterval = 5;
     lockCmd = "${pkgs.i3lock}/bin/i3lock -n -c AA0000";
+    lockCmdEnv = [ "DISPLAY=:0" ];
```

Comment:
Should be checking if this is generated below, if you're adding it to the test shape.

Reply:
added